### PR TITLE
Austin.intro.1.Arnold.Wells

### PR DESCRIPTION
**Pull request by Arnold73**

### CODE REVIEW

File: 02Lesson/01Project/index.html [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+    <title>
+      Arnold Wells Photography
+    </title>
+    <meta charset="UTF-8">
+  </head>
+  <body style="font-family: Arial">
+
+    <img src="https://scontent-dft4-2.xx.fbcdn.net/v/t1.0-9/10930907_905943542773873_7473205960492472388_n.jpg?oh=c1863cdb875abfe3f45b05069125126e&oe=589D2EAE" alt="Smiley face" width="500"/>
+    <hr>
+  <h1>
```
Comment (ivome) [inline 02Lesson/01Project/index.html:13]: Indentation: After self closing tag, the next opening tag should start at the same level.

File: 02Lesson/01Project/index.html [hunk 2]
```diff
@@ -1,9 +1,44 @@
...
+    Arnold Is An Awesome Photographer!
+  </h1>
+
+  <p>
+    Arnold provides entertainment and editorial photographic services for all
+    types of publications and corporate events.  Arnold Will always get the shot!
+
+    Published
+  </p>
+    <ul>
```
Comment (ivome) [inline 02Lesson/01Project/index.html:23]: Indentation: After closing tag, the next opening tag should be on the same level.

File: 02Lesson/02Project/index.html [hunk 3]
```diff
@@ -1,10 +1,55 @@
...
+        </h3>
+      </aside>
+    </main>
+    <footer>
+      <strong>
+          © The Daily Planet 2016
+      </strong>
+
+    </footer>
+  </body>
```
Comment (ivome) [inline 02Lesson/02Project/index.html:54]: Great formatting here!

**Review by ivome — COMMENTED** [review]
Great work besides some minor indentation issues. If you fix the indentation issues, the testsuite should also run through on github. 
Keep it up!

File: 02Lesson/01Homework/index.html [hunk 4]
```diff
@@ -1,8 +1,81 @@
...
+        <a href="https://www.flickr.com/photos/liverdog/" style="font-family:Arial">Flickr</a>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+          <h2 style="font-family: Arial">
+            Arnold Patterson Wells
+          </h2>
+          <img src="/01Lesson/HeadshotArnoldWells-3.jpg" alt="Arnold Wells" height="200">
```
Comment (zacacollier) [inline 02Lesson/01Homework/index.html]: move the image into your current directory, then use ``

**Pull request closed without merging**